src/main/java/com/ylysenkova/movieland/web/controller/MovieController.java

```diff
@@ -115,6 +112,17 @@
         }
     }
 
+    @RequestMapping(value = "/movie/{movieId}", method = RequestMethod.GET)
+    public @ResponseBody ResponseEntity<?> getMovieById (@PathVariable(value = "movieId") int movieId) {
+        logger.debug("Sending request ... ");
+        long startTime = System.currentTimeMillis();
+        Movie movie = movieService.getMovieById(movieId);
+        MovieWithReviewResponse  movieWithReviewResponse = new MovieWithReviewResponse(movie);
+
+        logger.debug("Movie {} is received.It took {} ms", movie, System.currentTimeMillis() - startTime);
```

Comment:
Why do you use debug instead info? Do you expect logging only for developers purposes? Because if on prod env will be set info level you will not have log at all.